### PR TITLE
Arrivals APC and obstruction removal + Chapel/chapel maint

### PR DESCRIPTION
**Pull request by Pyko1**
Removed excess APC at arrivals and made the lanes a bit less
obstructive.

capture

### CODE REVIEW

**Comment by Nokington** [comment]
:+1:

**Comment by Pyko1** [comment]
Made the russian spyroom its own area, and fixed the intercomms.

**Comment by Pyko1** [comment]
Some changes around chapel and maintenance next to it.

capture

**Comment by Nokington** [comment]
:+1:

**Comment by Pyko1** [comment]
Better picture of the changes in chapel area.

capture

**Comment by Spacedong** [comment]
you're putting 4 vending machines in a chapel, I don't understand, cig machine I get, you need lighters, but why soda, food, and coffee? its not a supermarket fam, put the vending machines in bar/hallway outside the chapel
???????????????????????????????????????????????????????

**Comment by Pyko1** [comment]
The back looked a bit too empty, and i kinda wish they would attract people occasionally in chapel, to witness chaplain cremating captain, and assistants spacing the clown. Currently chapel is just a place you walk through when someone has welded the firelocks on hallway.

File: html/changelogs/Pyko - Arrivals chapel APC.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "Made changes around chapel and maintenance next to it."
```
Comment (nalarmg) [inline html/changelogs/Pyko - Arrivals chapel APC.yml]: What changes?

**Comment by Pyko1** [comment]
Added matching shoes to colored wardrobes.
Also added more cloth options to arrivals shuttle.

**Comment by Nokington** [comment]
:+1:

**Comment by Lawligagger** [comment]
Alright

**Pull request merged**